### DISCUSSION TITLE
Support for Right-to-Left Direction in Admin Panel

### DISCUSSION THREAD

**Discussion by obezzad** [directus/directus#14005] [Feature Requests]
Hello Directus team and contributors,

When I try to use a RTL language, the interface stays LTR, which is unintuitive.

The Custom CSS is not enough for this without a great hassle as shown below. Meanwhile, is there any workaround to customize the core UI and styling of the admin panel?

![image](https://user-images.githubusercontent.com/2561446/174643397-939e2fbc-3eeb-46c5-ad70-86b72ce12625.png)

**Comment by rijkvanzanten** [comment]
The custom CSS is the only workaround to customize the core UI at the moment.

**Reply by benhaynes** [reply]
Just to add to this, we're aware of the RTL limitation and to handle this the _right_ way, we are taking additional R&D time to understand how this can be implemented in a way that will keep the codebase maintainable. We'll get this supported, but it may take some time... or require some community help.

**Reply by obezzad** [reply]
Thank you for your responsiveness and continuous support!

I would be glad to contribute to this issue whenever possible, I would just need some guidance regarding the current constraints.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
